### DISCUSSION TITLE
Zoom bar on plots with scaleType=linear

### DISCUSSION THREAD

**Discussion by diogorloureiro** [carbon-design-system/carbon-charts#1271] [Q&A]
Hello everyone,

I would like to use the Zoom Bar component in one of my plots with `"scaleType": "linear"`. However, whenever I try to add the Zoom Bar the plot just breaks. I also noticed that all the [Zoom Bar examples in Storybook](https://charts.carbondesignsystem.com/vue/?path=/story/utility-zoom-bar--line-time-series-time-interval-15seconds-zoom-bar-enabled) are using `"scaleType": "time"`, is there any reason for this?

Is it possible to use a Zoom Bar with scaleType=linear or will it be possible in a near future?

Thanks in advance,
Diogo

**Comment by theiliad** [comment]
yes unfortunately we only support the zoombar with our time-scale axes

**Reply by migatt** [reply]
Are there any plans to add the zoombar for linear axes in the future?
